### DISCUSSION TITLE
Requests/ADALM-PLUTO SDR

### DISCUSSION THREAD

**Discussion by ghost** [FutureSDR/FutureSDR#146] [Ideas]
Will the PlutoSDR be implemented similar to: FutureSDR/FutureSDR/tree/main/src/blocks/wasm/hack_rf.rs

**Comment by bastibl** [comment]
The HackRF driver that you linked is a WebUSB/WASM experiment. You can use the Pluto w/ Soapy drivers. If you have a lot of Soapy drivers installed and it picks the wrong one, you can specify the driver w/ `-a soapy_driver=pluto` (or plutosdr, not sure how Soapy calls the driver).
